src/plugins/blocksmith/components/instagram-media-block-preview/index.jsx

```diff
@@ -1,17 +1,90 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Message from '@gdbots/pbj/Message';
-import InstagramEmbed from 'react-instagram-embed';
 
-const InstagramMediaBlockPreview = ({ block }) => (
-  <InstagramEmbed
-    url={`https://instagr.am/p/${block.get('id')}/`}
-    hideCaption={block.get('hidecaption') || false}
-  />
-);
+export default class InstagramPostBlockPreview extends Component {
+  static propTypes = {
+    block: PropTypes.instanceOf(Message).isRequired,
+  };
 
-InstagramMediaBlockPreview.propTypes = {
-  block: PropTypes.instanceOf(Message).isRequired,
-};
+  constructor(props) {
+    super(props);
+    this.embedParentRef = React.createRef();
+    this.clean = this.clean.bind(this);
+    this.embed = this.embed.bind(this);
+    this.state = {
```

Comment:
this component likely doesn't need state, can we do this without

Reply:
State was removed